plugins/out_stackdriver/stackdriver.c

```diff
@@ -2942,15 +2942,18 @@ static void cb_stackdriver_flush(struct flb_event_chunk *event_chunk,
 #endif
           if (c->resp.status >= 400 && c->resp.status < 500) {
             ret_code = FLB_ERROR;
-            flb_plg_warn(ctx->ins, "error: %s", c->resp.payload);
+            flb_plg_warn(ctx->ins, "error for tag=%s: %s", event_chunk->tag,
```

Comment:
That's good, slight adjustment to the ordering:
```suggestion
            flb_plg_warn(ctx->ins, "tag=%s error sending to Cloud Logging: %s", event_chunk->tag,
```

This applied to each of the messages. The convention is for the `tag=%s` to usually go right after the plugin/log level designations.